web-admin/src/routes/[organization]/+page.ts

```diff
@@ -0,0 +1,21 @@
+import type { V1OrganizationPermissions } from "@rilldata/web-admin/client";
+import { checkUserAccess } from "@rilldata/web-admin/features/authentication/checkUserAccess";
+import { fetchOrganizationPermissions } from "@rilldata/web-admin/features/organizations/selectors";
+import { error } from "@sveltejs/kit";
+
+export const load = async ({ params: { organization } }) => {
+  let organizationPermissions: V1OrganizationPermissions = {};
+  if (organization) {
+    try {
+      organizationPermissions =
+        await fetchOrganizationPermissions(organization);
```

Comment:
Since this is fetched in the route `layout.ts` file, it can be accessed like: `const parentData = await parent();`